eventing/samples/github-events/README.md

```diff
@@ -0,0 +1,162 @@
+# GitHub Flow
+
+A GitHub webhook will be created on a repository and a Knative `Service` will be 
+deployed to receive the webhook's event deliveries and forward them into a 
+`Channel`, through a `Bus`, and out to the consumer via a `Subscription`. The 
+`Flow` resource takes care of provisioning the webhook, the `Service`, the 
+`Channel`, and the `Subscription`.
+
+In response to a pull request event, the sample app _legit_ Service will add 
+`(looks pretty legit)` to the PR title.
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative serving installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com/) installed and running on your local machine,
+  and a Docker Hub account configured (you'll use it for a container registry).
+- The Kubernetes cluster also has Knative eventing core installed. You can
```

Comment:
Suggestion: "The Kubernetes cluster also has Knative eventing core installed." --> "Knative eventing core installed on your Kubernetes cluster."